src/view/mainwidget.cpp

```diff
@@ -1,22 +1,50 @@
 #include "mainwidget.h"
+#include "persistence/persistence.h"
+#include <QDebug>
 
 
 MainWidget::MainWidget(QWidget *parent) : QWidget(parent)
 {
-    model_ = new Model();
+    p_ = new Persistence();
+    model_ = new Model(p_);
+    VBoxLayout_ = new QVBoxLayout();
     create_menubar();
+    list_ = new QListWidget();
+    list_->setVisible(false);
+    VBoxLayout_->addWidget(list_);
+    this->setFixedSize(600,600);
 }
 
 void MainWidget::create_menubar()
 {
 
-        QVBoxLayout* boxLayout = new QVBoxLayout(this); // Main layout of widget
+        //QVBoxLayout* boxLayout = new QVBoxLayout(this); // Main layout of widget
+        this->setLayout(VBoxLayout_);
 
         QMenuBar* menu_bar = new QMenuBar();
-        menu_bar->addAction("My Profile", this, SLOT(on_my_profile()));
+        /*menu_bar->addAction("My Profile", this, SLOT(on_my_profile()));
         menu_bar->addAction("Edit Quiz", this, SLOT(on_edit_quiz()));
-        menu_bar->addAction("Load Existing Project", this, SLOT(on_load_existing_project()));
+        menu_bar->addAction("Load Existing Quizzes", this, SLOT(on_load_existing_project()));
+        */
 
+        menu_bar->addAction("My Profile");
+        menu_bar->addAction("Edit Quiz");
+        auto *load_quiz_clicked = menu_bar->addAction("Load Existing Quizzes");
+        connect(load_quiz_clicked, &QAction::triggered,this,[=]()
+        {
```

Comment:
This list filling sould be in its own function.

Reply:
We should tlak about this further.